habitTrackerApp.py

```diff
@@ -24,18 +24,17 @@
 from kivy.core.window import Window
 from kivy.uix.checkbox import CheckBox
 
-Window.left = -1000
+# Window.left = -1000
 Window.top = 50
 Window.size = (800,600)
+Window.clearcolor = (1, 1, 1, 1)
```

Comment:
sometimes it good to add in which unit those number
I guess by experience it's pixel then screen resolution and rgb